desktop_version/src/Render.cpp

```diff
@@ -623,6 +623,13 @@ static void menurender(void)
             else
                 graphics.Print(-1, 65, "Fake loading screen is ON", tr, tg, tb, true);
             break;
+        case 4:
+            graphics.bigprint(-1, 30, "In Game Timer", tr, tg, tb, true);
+            if (game.showingametimer)
```

Comment:
Should probably have braces around these single-line statements.

Reply:
gotcha, the one above it didn't have braces too so I just did it like that